### PR TITLE
Adding a department model

### LINKED ISSUE #64: Department
Model to encapsulate the department
- Department Name
- Department Head
- Admin Assistant

### PR DESCRIPTION
**Pull request by carolyncole**
Added in todo for connecting to StaffProfile once that model exists

closes #64

### CODE REVIEW

File: app/assets/javascripts/departments.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (hackmastera) [inline app/assets/javascripts/departments.coffee]: I've been deleting these coffee files; I don't think we use coffeescript anywhere else?

File: app/assets/stylesheets/departments.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Departments controller here.
```
Comment (hackmastera) [inline app/assets/stylesheets/departments.scss]: delete?

File: app/helpers/departments_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module DepartmentsHelper
```
Comment (hackmastera) [inline app/helpers/departments_helper.rb]: delete?

File: spec/helpers/departments_helper_spec.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+# the DepartmentsHelper. For example:
+#
+# describe DepartmentsHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe DepartmentsHelper, type: :helper do
```
Comment (hackmastera) [inline spec/helpers/departments_helper_spec.rb]: delete this spec

File: spec/models/department_spec.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+require "rails_helper"
+
+RSpec.describe Department, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (hackmastera) [inline spec/models/department_spec.rb]: For other models I've used this spec as a place to list all attributes it should respond to. I think that's a nice convention; what do you think?

**Comment by carolyncole** [comment]
@hackmastera Thanks for the review.  Good to keep things consistent.  All requests have been updated.

**Pull request merged**